conans/client/cmd/new.py

```diff
@@ -11,8 +11,10 @@ class {package_name}Conan(ConanFile):
     name = "{name}"
     version = "{version}"
     license = "<Put the package license here>"
+    author = "<Put your name here> <And your email here>"
     url = "<Package recipe repository url here, for issues about the package>"
     description = "<Description of {package_name} here>"
+    tags = ["<Put some tag here>", "<here>", "<and here>"]
```

Comment:
A bit asymmetric compared to settings and other things. Should we use the tuple as default way? Maybe as the unique way (maybe support defining a single string too)?

Having multiple input formats introduces a bit more complexity, it has already bitten us in the past for other attributes, maybe we could be a bit more opinionated here this time @lasote?

Reply:
I used the same approach from license attribute, but tuple could be nice.

both string, tuple, list are allowed. I could change to tuple and add more tests to exercise that situation.

Reply:
updated to `topics: ('foo', 'bar', 'qux')`